src/Users/Client.java

```diff
@@ -0,0 +1,70 @@
+package Users;
+
+import Book.Book;
+import Book.Author;
+import Book.BookType;
+import Users.User;
+import org.jetbrains.annotations.NotNull;
+
+import java.time.LocalDateTime;
+import java.time.format.DateTimeFormatter;
+import java.util.*;
+
+public class Client extends User {
+    private String phoneNumber;
+    private HashMap<String, List<Book>> rentedBooks = new HashMap<String, List<Book>>();
+    private Set<Book> favouriteBooks = new HashSet<Book>();
+    private List<Author> favouriteAuthors = new ArrayList<Author>();
```

Comment:
use just the diamond operator, will infer the type: 
=new HashMap<>() 
=new HashSet<>();
=new ArrayList<>()